.github/workflows/ci/common_defs/toolchain-ppc-vxworks6.9.cmake

```diff
@@ -0,0 +1,61 @@
+# This example toolchain file describes the cross compiler to use for
+# the target architecture indicated in the configuration file.
+
+# Basic cross system configuration
```

Comment:
There are a bunch of toolchain files that look like they're targeted at platforms we're never going to run these examples on. If we need these files, lets just include the one for PC linux.

Reply:
Removed the extra toolchain files but needed to keep the cpu ones